src/shared/components/Resources/ResourcePreview.tsx

```diff
@@ -28,12 +28,7 @@ const ResourcePreview: React.FunctionComponent<
 
   if (!file) {
     // TODO: Refactor after ES Lists are fetched by Type
-    NexusFile.getSelf(
-      resource.self,
-      resource.orgLabel,
-      resource.projectLabel,
-      true
-    )
+    NexusFile.getSelf(resource.self, true)
```

Comment:
Is your component calling the API and fetching stuff by itself? :scream:

Reply:
🥺